test/browser/index.js

```diff
@@ -1,3 +1,2 @@
 'use strict';
-require('./custom-elements.js');
```

Comment:
Ah. Yes. I wanted to remove custom elements, but then it turned out to be more cumbersome than imagined. I had already removed this test. I should probably have put it back in, since custom elements are still available. Good catch.